### PR TITLE
Improve Documentation

### PR DESCRIPTION
**Pull request by mpahl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,36 @@
 # collect_stata
 
-Accumulate data from stata files and write it into an open format 
+The tool reads dta-files, calculate some univariate statistics for each variable 
+and writes out a json-file with these calculations and the metadata.
 
-# Setup for development
+The purpose of the project is the preprocessing of the soep data for ddionrails.
```
Comment (afuetterer) [inline README.md]: Please remove soep here, as this tool is useful for a wider range.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,36 @@
 # collect_stata
 
-Accumulate data from stata files and write it into an open format 
+The tool reads dta-files, calculate some univariate statistics for each variable
```
Comment (afuetterer) [inline README.md]: calculates

File: README.md [hunk 3]
```diff
@@ -1,13 +1,36 @@
 # collect_stata
 
-Accumulate data from stata files and write it into an open format 
+The tool reads dta-files, calculate some univariate statistics for each variable 
+and writes out a json-file with these calculations and the metadata.
```
Comment (afuetterer) [inline README.md]: What is "metadata" in this case?

Comment (mpahl) [reply README.md]: extracted from stata - the description of the given data

**Comment by afuetterer** [comment]
Good job rebasing your branch and squashing the commits, @mpahl.

One more thing, now that you know how to rebase and squash, please have a look at "reword",
see https://help.github.com/en/articles/changing-a-commit-message

It is good that you reference the issues by their id with `see #33`. But it be even better if you add
`[close #33]`
`[fix #37]`
see https://help.github.com/en/articles/closing-issues-using-keywords

Also it would be nice to have one PR per fixed issue.
As this branch and PR are both called "docstrings edit", I assumed it would be include changes to some docstrings. Right now, this is not the case.

Please stop pushing to this branch, so we can merge it soon. Consider opening a new branch per issue, if it makes sense.

File: .gitignore [hunk 4]
```diff
@@ -102,3 +102,4 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+.vscode/settings.json
```
Comment (hansendx) [inline .gitignore]: I would consider this an issue of ones local dev environment.
Please put files to be ignored in your local repository into `.git/info/exclude`.
I would also recommend to ignore the whole folder.

File: README.md [hunk 5]
```diff
@@ -1,13 +1,36 @@
 # collect_stata
 
-Accumulate data from stata files and write it into an open format 
+The tool reads dta-files, calculates some univariate statistics for each variable 
+and writes out a json-file with these calculations and the metadata (extracted from stata).
 
-# Setup for development
+The purpose of the project is the preprocessing of data for ddionrails.
```
Comment (hansendx) [inline README.md]: Please create a link to the ddionrails repository. 
```markdown
... data for [ddionrails](https://github.com/ddionrails/ddionrails)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
```
Comment (hansendx) [inline CHANGELOG.md]: An "unreleased" link would be very nice.

```markdown
[unreleased](https://github.com/ddionrails/collect_stata/compare/v0.0.1...develop)
```

File: collect_stata/read_stata.py [hunk 7]
```diff
@@ -7,8 +7,6 @@
 
 import pandas as pd
 
-LOGGER = logging.getLogger(__name__)
```
Comment (hansendx) [inline collect_stata/read_stata.py]: This change does not seem to belong into this branch/PR.

**Comment by hansendx** [comment]
> Good job rebasing your branch and squashing the commits, @mpahl.
> 
> One more thing, now that you know how to rebase and squash, please have a look at "reword",
> see https://help.github.com/en/articles/changing-a-commit-message
> 
> It is good that you reference the issues by their id with `see #33`. But it be even better if you add
> `[close #33]`
> `[fix #37]`
> see https://help.github.com/en/articles/closing-issues-using-keywords
> 
> Also it would be nice to have one PR per fixed issue.
> As this branch and PR are both called "docstrings edit", I assumed it would be include changes to some docstrings. Right now, this is not the case.
> 
> Please stop pushing to this branch, so we can merge it soon. Consider opening a new branch per issue, if it makes sense.

I would agree but I would add, that you should stop pushing to this branch only after the changes from the review are addressed.

**Comment by afuetterer** [comment]
Is this ready to be merged yet?

**Comment by mpahl** [comment]
yes

**Pull request merged**